### DISCUSSION TITLE
how to learn   sqlalchemy2.0

### DISCUSSION THREAD

**Discussion by techUdayMungalpara** [sqlalchemy/sqlalchemy#10381] [Usage Questions]
i try to learn from sqlalchemy 2.0 as biggner from couple of days but
 i am not getting things clear and try to find resource on udemy , youtube video but not able to find the solution if you can provide any resource or github repo link . 
Thank you

**Comment by zzzeek** [comment]
hi, I saw your comment on #8505.   Within the SQLAlchemy project, the docs at https://docs.sqlalchemy.org/en/20/ are what we have.  There are lots and lots of third party tutorials, videos, and also chatgpt can be used.   There's nothing for me to share here other than what I had in my comment at https://github.com/sqlalchemy/sqlalchemy/discussions/8505#discussioncomment-3616240 , which I understand may not be enough for every kind of user, so I would advise reaching out for other kinds of tutorials on the web, there's a bunch.
